src/components/HolidayRecommendationListModal.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Modal from '@material-ui/core/Modal';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  modal: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    margin: '1.5rem'
+  },
+  modalBody: {
+    padding: '1rem',
+    backgroundColor: '#fff'
+  }
+}));
+
+const HolidayRecommendationListModal = ({
+  isOpen,
+  handleClose,
+  holidayRecommendations
+}) => {
+  const classes = useStyles();
+
+  const topFiveHolidayRecommendations =
+    holidayRecommendations.length > 5
+      ? holidayRecommendations.slice(0, 5)
```

Comment:
You can leave out the check for length > 5 since `slice` will simply return shallow copy with all the elements if the array has less than 5 elements anyway. 🙂